test/OrchardCore.Tests/Apis/Context/SiteContext.cs

```diff
@@ -30,6 +29,10 @@ public virtual async Task InitializeAsync()
             await semaphoreSlim.WaitAsync();
             try
             {
+                if (DefaultTenantClient == null)
+                {
+                    DefaultTenantClient = Site.CreateClient();
```

Comment:
Remove `CreateClient` altogether, the httpclient should a single static instance.

Reply:
Sorry, not sure i understand. First a client needs to be created from the `Site` which is a `WebApplicationFactory` and it can't be used concurrently.

Then because the part which creates and setups a tenant is not run concurrently we call `WebApplicationFactory.CreateClient()` to init an unique static field for this part.

Then, unless the above part, the rest is done concurrently. That's why below we also call `Client = Site.CreateDefaultClient(url)` to have a separate client for the rest of the work of each testing class.